doc/src/cylc-user-guide/cug.tex

```diff
@@ -4604,13 +4604,19 @@ \subsubsection{Parameter Expansion}
 
 \paragraph{Zero-Padded Integer Values}
 
-Integer parameter values are zero-padded according to the size of their
-largest value, so \lstinline@foo<p>@ for \lstinline@p = 9..10@ expands to
+Integer parameter values are, by default, zero-padded according to the size of
+their largest value, so \lstinline@foo<p>@ for \lstinline@p = 9..10@ expands to
 \lstinline@foo_p09, foo_p10@.
 
-To get thicker padding, prepend extra zeroes to the upper range value:
-\lstinline@foo<p>@ for \lstinline@p = 9..010@ expands to
-\lstinline@foo_p009, foo_p010@.
+To get thicker padding and/or alternate suffixes, use a template. E.g.:
+
+\begin{lstlisting}
+[cylc]
+    [[parameters]]
+        p = 3..14
+    [[parameter templates]]
+        p = %%p%(p)03d  # suffixes = %p003, %p004, ..., %p013, %p014
```

Comment:
I think this may be clearer without the escaped `%`:

```ini
p = p%(p)03d  # suffixes = p003, p004, ..., p013, p014
```

Reply:
In which case, I'll add another example. (I think it will still be useful to have an example on how to have a `%` in your template.)